"2019-08-23 09:44:26,516 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:44:26,516 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:44:26,520 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:44:26,520 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:44:37,590 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:44:37,591 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:44:38,100 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:44:38,100 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:44:49,136 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:44:49,136 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:44:49,579 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:44:49,579 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:45:00,436 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:45:00,436 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:45:00,450 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:45:00,450 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:45:11,467 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:45:11,467 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:45:12,213 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:45:12,213 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:45:14,673 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/b3d612c8-c58a-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 09:45:23,124 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:45:23,124 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:45:23,694 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:45:23,694 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:45:26,465 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-23 09:45:35,038 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:45:35,038 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:45:35,041 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:45:35,041 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:45:46,310 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:45:46,310 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:45:46,896 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:45:46,897 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:45:57,792 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:45:57,792 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:45:57,806 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:45:57,806 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:46:00,638 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/cf09bcde-c58a-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 09:46:09,007 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:46:09,007 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:46:09,015 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:46:09,015 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:46:20,233 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:46:20,233 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:46:20,777 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:46:20,777 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:46:31,681 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:46:31,681 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:46:31,832 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:46:31,833 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:46:43,000 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:46:43,000 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:46:43,004 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:46:43,005 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:46:54,421 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:46:54,421 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:46:54,966 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:46:54,966 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:46:57,797 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 09:47:06,515 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:47:06,515 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:47:07,044 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:47:07,044 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:47:18,604 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:47:18,605 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:47:18,613 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:47:18,613 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:47:21,220 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 09:47:29,875 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:47:29,875 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:47:29,932 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:47:29,932 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:47:35,908 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:47:35,908 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:47:35,930 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:47:35,931 - data_overview.py[line:90] - WARNING: destroy Dtable error,:'NoneType' object has no attribute 'destroy', but this can be ignored sometimes"
"2019-08-23 09:47:50,169 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/1076e57a-c58b-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 09:49:09,570 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/3f7ffbcc-c58b-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 09:50:41,689 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/76a249f2-c58b-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 09:50:53,057 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 09:51:22,098 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/8e2c6c6a-c58b-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 09:54:23,158 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 09:54:34,481 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/0159c08e-c58c-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 09:54:56,858 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 09:56:04,457 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 09:56:50,900 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/52324d32-c58c-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 09:57:02,200 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-23 09:57:24,631 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/66e20254-c58c-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 09:57:41,740 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 09:58:04,525 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-23 09:58:49,206 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-23 09:59:23,282 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/ad4260fe-c58c-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:01:01,340 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:01:12,919 - data_overview.py[line:90] - WARNING: destroy Dtable error,:/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/ee85f65c-c58c-11e9-ad5e-0242ac110002/0: No such file or directory, but this can be ignored sometimes"
"2019-08-23 10:01:24,353 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:01:46,431 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/02b385a4-c58d-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:02:09,218 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:02:20,665 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/16c4aac8-c58d-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:02:55,175 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:03:18,161 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:03:41,819 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-23 10:04:16,827 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:04:22,584 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:05:53,070 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:07:14,223 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:09:03,063 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:10:11,772 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-23 10:10:22,947 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-23 10:11:03,008 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:11:25,860 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 39] Directory not empty: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/5c4a14ba-c58e-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:12:21,929 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-23 10:12:56,166 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:13:07,776 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-23 10:13:19,429 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-23 10:14:35,350 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:15:19,961 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:16:05,597 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:16:28,660 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/1019334a-c58f-11e9-ad5e-0242ac110002/0', but this can be ignored sometimes"
"2019-08-23 10:16:40,926 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/18108832-c58f-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:17:04,946 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:17:16,300 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 39] Directory not empty: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/2ce7664a-c58f-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:17:27,598 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-23 10:17:39,241 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-23 10:17:56,593 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:18:29,906 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-23 10:19:49,746 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:20:13,139 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-23 10:20:24,830 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:20:36,343 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-23 10:23:10,846 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-23 10:23:46,442 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:24:20,699 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/29c636de-c590-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:25:34,851 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-23 10:26:08,709 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/6a59f26c-c590-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:26:32,292 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/785e2e46-c590-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:27:48,949 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-23 10:28:22,495 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-23 10:28:33,707 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/c0895bb4-c590-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:28:56,832 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/cdeb425e-c590-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:29:08,165 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-23 10:29:19,898 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/dbe98852-c590-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:29:43,210 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/ea08ef9a-c590-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:30:05,720 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:31:03,884 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 39] Directory not empty: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/19fd0bdc-c591-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:31:55,376 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 39] Directory not empty: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/38992e2c-c591-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:32:06,831 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:32:29,753 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/4d39ccb0-c591-11e9-ad5e-0242ac110002/0', but this can be ignored sometimes"
"2019-08-23 10:32:41,451 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/54722dc4-c591-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:32:53,013 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/5b672db4-c591-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:33:27,899 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-23 10:34:03,386 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-23 10:34:15,009 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/8c4c40b8-c591-11e9-ad5e-0242ac110002/0', but this can be ignored sometimes"
"2019-08-23 10:34:26,351 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:35:40,326 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/bea5db32-c591-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:35:51,724 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/c5a19d5e-c591-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:36:03,107 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 39] Directory not empty: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/cc53ddc4-c591-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:36:26,061 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/da29a942-c591-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:36:49,205 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:37:01,526 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:37:37,035 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-23 10:38:16,909 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:39:02,718 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/37ae4848-c592-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:39:14,054 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 39] Directory not empty: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/3e300116-c592-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:39:25,298 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/45260f60-c592-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:39:37,288 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-23 10:39:48,610 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-23 10:40:00,042 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:40:46,884 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:40:58,740 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:41:10,169 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-23 10:41:49,961 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-23 10:42:01,385 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-23 10:42:47,172 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/bd18d354-c592-11e9-ad5e-0242ac110002/0', but this can be ignored sometimes"
"2019-08-23 10:43:21,990 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/d1ba11a6-c592-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:43:44,912 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-23 10:44:20,378 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/f4964910-c592-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:45:00,943 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-23 10:45:35,085 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-23 10:45:46,512 - data_overview.py[line:90] - WARNING: destroy Dtable error,:/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/2835a496-c593-11e9-ad5e-0242ac110002/0: No such file or directory, but this can be ignored sometimes"
"2019-08-23 10:45:57,553 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/2ecfabe4-c593-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:46:20,628 - data_overview.py[line:90] - WARNING: destroy Dtable error,:/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/3ccf10b8-c593-11e9-ad5e-0242ac110002/0: No such file or directory, but this can be ignored sometimes"
"2019-08-23 10:47:18,340 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-23 10:47:30,349 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'data.mdb', but this can be ignored sometimes"
"2019-08-23 10:47:42,146 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/6d433346-c593-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:47:59,591 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
"2019-08-23 10:48:44,682 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/923ab0ca-c593-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:49:29,856 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '0', but this can be ignored sometimes"
"2019-08-23 10:49:41,370 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/b43bb886-c593-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:49:53,377 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/bb687842-c593-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:50:04,652 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 39] Directory not empty: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/c23bf66c-c593-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:50:16,099 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: '/fate/data/IN_MEMORY/hetero_network_embedding_example_standalone_20190823094402/c8bae2fa-c593-11e9-ad5e-0242ac110002', but this can be ignored sometimes"
"2019-08-23 10:51:20,154 - data_overview.py[line:90] - WARNING: destroy Dtable error,:[Errno 2] No such file or directory: 'lock.mdb', but this can be ignored sometimes"
